statuscake/resource_statuscaketest.go

```diff
@@ -308,24 +314,19 @@ func ReadTest(d *schema.ResourceData, meta interface{}) error {
 	d.Set("port", testResp.Port)
 	d.Set("trigger_rate", testResp.TriggerRate)
 	d.Set("custom_header", testResp.CustomHeader)
-	d.Set("user_agent", testResp.UserAgent)
 	d.Set("status", testResp.Status)
 	d.Set("uptime", testResp.Uptime)
-	if err := d.Set("node_locations", testResp.NodeLocations); err != nil {
+	if err := d.Set("node_locations", considerEmptyStringAsEmptyArray(testResp.NodeLocations)); err != nil {
 		return fmt.Errorf("[WARN] Error setting node locations: %s", err)
 	}
-	d.Set("ping_url", testResp.PingURL)
```

Comment:
Why are you removing all these parameters?

Reply:
Because the detail response from statuscake doesn’t appear to ever contain them.

So all I’ve seen these statements do is clobber the state with an empty value.

Reply:
Oh damn, you can _update_ these parameters, but you cannot get it with detailed data? Are they serious?!